horizon/backend/api/v1/router/namespaces.py

```diff
@@ -146,7 +145,8 @@ async def get_namespace_permissions(
         permissions_dict = await unit_of_work.namespace.get_namespace_users_permissions(namespace_id)
 
         permissions_response = [
-            PermissionResponseItemV1(username=user.username, role=role.name) for user, role in permissions_dict.items()
+            PermissionResponseItemV1(username=user.username, role=role.name)  # type: ignore[arg-type]
```

Comment:
nit: could we just convert the role value to proper enum?

Reply:
Change it here or at separate PR?

Reply:
It just 2 lines here and below